builtin/providers/google/resource_storage_bucket.go

```diff
@@ -295,3 +338,41 @@ func resourceStorageBucketStateImporter(d *schema.ResourceData, meta interface{}
 	d.Set("name", d.Id())
 	return []*schema.ResourceData{d}, nil
 }
+
+func expandCors(configured []interface{}) []*storage.BucketCors {
+	corsRules := make([]*storage.BucketCors, 0, len(configured))
+	for _, raw := range configured {
+		data := raw.(map[string]interface{})
+		corsRule := storage.BucketCors{}
```

Comment:
style nit: totally just my opinion but I think filling out structs/maps inline looks a bit better:
```
		corsRule := storage.BucketCors{
			Origin:         convertSchemaArrayToStringArray(data["origin"].([]interface{})),
			Method:         convertSchemaArrayToStringArray(data["method"].([]interface{})),
			ResponseHeader: convertSchemaArrayToStringArray(data["response_header"].([]interface{})),
			MaxAgeSeconds:  int64(data["max_age_seconds"].(int)),
		}
```
and below:
```
		data := map[string]interface{}{
			"origin":          corsRule.Origin,
			"method":          corsRule.Method,
			"response_header": corsRule.ResponseHeader,
			"max_age_seconds": corsRule.MaxAgeSeconds,
		}
```